_pages/index.html

```diff
@@ -7,9 +7,9 @@
   - owl.theme.default.min
 ---
 <div id="index-jumbotron" class="jumbotron text-center pt-0">
-    <img src="{{ site.baseurl }}/static/img/compsoc-profile-white-nomargin.png" class="img-fluid" alt="CompSoc Edinburgh Logo"
-    style="max-height: 300px; padding: 40px 30px">
-
+   <div style="overflow-y: hidden; max-height: 300px"> 
+      <img src="{{ site.baseurl }}/static/img/compsoc-profile-white-fill.png" class="img-fluid" alt="CompSoc Edinburgh Logo" style="margin-top: -40px; max-height: 400px" /> 
```

Comment:
Should there be a `max-height` on the parent `div` as well as the `img` itself?

Reply:
parent div removes image padding without needing to crop it by being shorter than the image and having `overflow-y: hidden`; img has negative margin to accomplish this without losing vertical center(-ish) align